opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -888,4 +888,8 @@ org.opennms.newts.nan_on_counter_wrap=true
 # By default this option is disabled.
 # org.opennms.features.telemetry.ingressAndEgressRequired=false
 
+# The use of multiple Minions in one location can break the alarm life-cycle for a some OpenNMS features.
```

Comment:
Let's avoid any changes in this file since we're targeting `foundation-2020`.

Reply:
Ok. I removed the entry.